phpstan.neon

```diff
@@ -44,3 +44,7 @@ parameters:
         - '#Method Symfony\\Component\\Serializer\\Normalizer\\AbstractObjectNormalizer::__construct\(\) invoked with 6 parameters, 0-4 required\.#'
         - '#Parameter \#1 \$bitmask of class Symfony\\Component\\Serializer\\Encoder\\JsonEncode constructor expects int, array<string, int> given\.#'
         - '#Parameter \#1 \$associative of class Symfony\\Component\\Serializer\\Encoder\\JsonDecode constructor expects bool, array<string, true> given\.#'
+
+        # Expected, due to fixture detection in webonyx/graphql-php
+        - '#Parameter \#2 \$errors of class GraphQL\\Executor\\ExecutionResult constructor expects array<GraphQL\\Error\\Error>, array<int\|string, Exception\|string> given\.#'
```

Comment:
I'm not sure if this line:
https://github.com/api-platform/core/blob/master/src/GraphQl/Action/EntrypointAction.php#L67
is still working.
I think it should be interesting to add a test.

Reply:
indeed, removing it doesn't break any existing tests IIRC so I wasn't sure.
I'll dig into that